README.md

```diff
@@ -89,6 +91,14 @@ Links:
 Procedure:
 *TBA*
 
+### Current Release Cycle
+
+[Dates and further information for the 1.23 Release](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.23)
```

Comment:
Not a blocker: Since this link has to be changed every release cycle, do you think it might be better to just link the [releases](https://github.com/kubernetes/sig-release/tree/master/releases) instead?